packages/app/src/cli/models/extensions/specifications/flow_template.ts

```diff
@@ -24,36 +34,31 @@ const spec = createExtensionSpecification({
   appModuleFeatures: (_) => ['bundling'],
   deployConfig: async (config, extensionPath) => {
     return {
-      title: config.title,
-      handle: config.handle,
-      description: config.description,
-      uuid: config.handle,
-      categories: config.categories,
-      require_app: config.require_app,
-      visible: config.visible,
-      enabled: config.enabled,
-      localization: await loadLocalesConfig(extensionPath, 'flow_template'),
-      definition: await loadWorkflow(extensionPath, config.handle),
+      templates: await Promise.all(
+        config.templates.map(async (template) => {
+          return {
+            key: template.key,
+            name: template.name,
+            description: template.description,
+            categories: template.categories,
+            require_app: template.require_app,
+            listed: template.listed,
+            enabled: template.enabled,
+            definition: await loadWorkflow(extensionPath, template.key),
+            localization: await loadLocalesConfig(joinPath(extensionPath, template.key), template.key),
+          }
+        }),
+      ),
     }
   },
 })
 
-async function loadWorkflow(path: string, handle: string) {
-  const flowFilePaths = await glob(joinPath(path, '/template.flow'))
+async function loadWorkflow(path: string, key: string) {
+  const flowFilePaths = await glob(joinPath(path, key, `${key}.flow`))
```

Comment:
so each template will be in a different folder?
wouldn't it make more sense to have all the templates in the root of the extension next to the toml file? or in a `templates` folder, but 1 folder per template seems to much

Reply:
I'd also make this be an explicit config as well, set this path to be the default but allow a developer to specify the file if they want

Reply:
By separating everything out it became easier to handle multiple templates as we will ended up sending them one at a time through kafka. The .flow file along with the translations will be in that folder as well.

Reply:
We have to separate the translations also to keep size down in the future.

Reply:
I thought there was going to be just the `.flow` file, if the plan is to have more files per folder then that's 👍 for me